### ISSUE TITLE
[Feat]: Overwrite open PR

### ISSUE THREAD

**Issue by kdeldycke** [AndreasAugustin/actions-template-sync#482]
### Describe the feature

I'd like to have a way to tell `actions-template-sync` it is okay to overwrite an existing PR.

This might solve issue #481, but also goes beyond that, as it will reset the commits attached to that PR and force a push.

### Use Case

I want my workflows based on `actions-template-sync` to have authority on the PR it creates, but also gives `actions-template-sync` the permission to scrap any changes added after its run.

It will also address some edge-cases I stumble into while experimenting in the context #467, where my PR is stuck because `actions-template-sync` is not capable of getting over some merge conflicts (see #476).

### Acknowledgements

- [ ] I may be able to implement this feature request

**Comment by AndreasAugustin** [comment]
Hi @kdeldycke and thanks for your patience.
With #499 you have the option to force push PRs. Please note that you also need to add more permissions for the workflow (`repository-projects: read`)

```yaml
  permissions:
    contents: write
    pull-requests: write
    repository-projects: read
```

This will also close #481 . Decided to have the option to opt in to this feature (not forcing per default).

Will be available in `v1.12.0`

**Comment by kdeldycke** [comment]
Can confirm this is working! :)
